--- env/lib/python3.8/site-packages/timidity/__main__.py	2020-06-27 18:13:15.401938584 -0400
+++ env/lib/python3.8/site-packages/timidity/__main__new.py	2020-06-27 18:13:02.618609743 -0400
@@ -12,7 +12,7 @@
 
     i = 1
     for filename in midi_list:
-        print(f"{i}.) {filename}")
+        #print(f"{i}.) {filename}")
         i += 1
     select_idx = int(input("Select a file: ")) - 1
     selection = midi_list[select_idx]
@@ -24,7 +24,7 @@
     wavfile.write(f"wav/{basename}.wav", 44100, audio)
 
     player.wait_done()
-    print("Done!")
+    #print("Done!")
 
 
 if __name__ == "__main__":
