--- env/lib/python3.8/site-packages/timidity/player.py	2020-06-27 18:20:40.003118753 -0400
+++ env/lib/python3.8/site-packages/timidity/player_new.py	2020-06-27 18:20:56.046309319 -0400
@@ -20,19 +20,16 @@
     song = np.zeros_like(t)
 
     for idx, note in enumerate(notes):
-        print("\rArranging notes... %d / %d" % (idx + 1, len(notes)), end='')
         start_pos = np.searchsorted(t, to_seconds(note.start))
         end_pos = np.searchsorted(t, to_seconds(note.end))
 
         song[start_pos:end_pos] += wave(
             t[start_pos:end_pos] * get_freq(note.pitch) * 2 * np.pi) * (note.velocity / 127.)
 
-    print()
 
     song = song / np.max(song)
     audio = (song * (2 ** 15 - 1) * volume).astype(np.int16)
 
-    print("Playing song...")
     player = sa.play_buffer(audio, 1, 2, sample_rate)
 
     if wait_done:
